Modules/Map/QuestieMap.lua

```diff
@@ -328,27 +328,7 @@ function QuestieMap.ProcessQueue()
         minimapDrawCall[2]._loaded = true
         if minimapDrawCall[2]._needsUnload then
             minimapDrawCall[2]:Unload()
-        end
-    end
-end
-
-local weaponMasterSkills = {
-    -- Alliance Weapon Trainers
-    [11865] = {"Fist Weapons", "Guns", "One-Handed Axes", "One-Handed Maces", "Two-Handed Axes", "Two-Handed Maces"},
-    [11866] = {"Bows", "Daggers", "Fist Weapons", "Staves", "Thrown"},
-    [11867] = {"Crossbows", "Daggers", "One-Handed Swords", "Polearms", "Staves", "Two-Handed Swords"},
-    [13084] = {"Crossbows", "Daggers", "Thrown"},
-    -- Horde Weapon Trainers
-    [2704] = {"Bows", "One-Handed Axes", "Staves", "Thrown", "Two-Handed Axes"},
-    [11868] = {"Bows", "Daggers", "Fist Weapons", "One-Handed Axes", "Thrown", "Two-Handed Axes"},
-    [11869] = {"Guns", "One-Handed Maces", "Staves", "Two-Handed Maces"},
-    [11870] = {"Crossbows", "Daggers", "One-Handed Swords", "Polearms", "Two-Handed Swords"},
-}
-local function appendSkillsToTitle(title, skills)
-    for _, skill in ipairs(skills) do
-        title = title .. "\n - " .. l10n(skill)
-    end
-    return title
+        end  
```

Comment:
You removed one `end` too much, so the entire code is broken atm. Did you test this?

Reply:
For some reason, this was not the case locally. So very unsure how this has gotten in there. Anyways, it's now fixed.